src/components/input/input.scss

```diff
@@ -91,6 +91,12 @@ md-input-container {
     bottom: 100%;
     @include rtl(left, 0, auto);
     @include rtl(right, auto, 0);
+
+    &.md-required:after {
+      content: ' *';
```

Comment:
Asterisk will be aligned in the same direction as the other text, so it's compatible with RTL, LTR.
But it seems that the placeholders are aligned wrong? (See [here](http://codepen.io/DevVersion/pen/zraQxj).

But in general, Yes asterisk will be reversed for RTL text